falcon/bench/bench.py

```diff
@@ -254,11 +254,12 @@ def run(frameworks, trials, iterations, stat_memory):
 def main():
     frameworks = [
         'bottle',
+        'django',
         'falcon',
         'falcon-ext',
         'flask',
         'pecan',
-        'werkzeug'
+        'werkzeug',
```

Comment:
Oh, it's just an old stylistic habit of mine. It avoids problems when you rearrange lines or whatnot and have to remember to add the comma at that point.